functions/_common/readmeTools.ts

```diff
@@ -265,10 +269,10 @@ export function constructReadme(
 
     const downloadDate = formatDate(new Date()) // formats the date as "October 10, 2024"
     if (isSingleColumn)
-        readme = `# ${grapher.title} - Data package
+        readme = `# ${grapher.displayTitle} - Data package
```

Comment:
`grapher.title` is undefined for mdims. The title we show in the UI gets set after we fetch the indicator metadata, so we have to use `grapher.displayTitle`. This should be safe also for normal graphers, since we return `grapher.title` from `grapher.displayTitle`, if it exists.